bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -72,6 +78,19 @@ export default withRouter(injectIntl(withModalMounter(createContainer((
     },
   });
 
+  // forcelly logged out when the meeting is ended
+  Meetings.find({ meetingId: Auth.meetingID }).observeChanges({
+    removed(old) {
+      Auth.clearCredentials()
```

Comment:
https://github.com/JaeeunCho/bigbluebutton/blob/fd2b5926bc92f17819cacf0b0b142cc6aebf8b77/bigbluebutton-html5/imports/ui/components/app/container.jsx#L63-L69

You are duplicating the code can you move it to a function?